src/gui/application.cpp

```diff
@@ -246,8 +246,15 @@ Application::Application(int &argc, char **argv)
         return;
 
 #if defined(WITH_CRASHREPORTER)
-    if (ConfigFile().crashReporter())
-        _crashHandler.reset(new CrashReporter::Handler(QDir::tempPath(), true, CRASHREPORTER_EXECUTABLE));
+    if (ConfigFile().crashReporter()) {
+        auto reporter = QStringLiteral(CRASHREPORTER_EXECUTABLE);
+#ifdef Q_OS_WIN
+        if (reporter.endsWith(QLatin1String(".exe"))) {
+            reporter.append(QLatin1String(".exe"));
+        }
```

Comment:
I think this is the opposite of what you want to do  :trollface:

Reply:
AFAICT it should never have .exe anyway tho...

Reply:
your right the if is wrong 🙈 
But the crashreporter.exe is an exe so ....
if the full file name is passed it uses the exe close to the binary if available, else it falls back to path.

Reply:
Ah yes we don't append exe yet, but ...